index.js

```diff
@@ -19,6 +27,7 @@ export default ({
   activeOpacityFeedback = 0.3,
   titleProps = {},
   titleStyle = {},
+  testID = "",
```

Comment:
testID is not a good name.
rename to `touchableWrapperProps` .
also add `touchableWrapperStyle` properly.